src/autoformat.js

```diff
@@ -54,7 +54,7 @@ export default class Autoformat extends Plugin {
 
 		if ( commands.get( 'numberedList' ) ) {
 			// eslint-disable-next-line no-new
-			new BlockAutoformatEngine( this.editor, /^\d+[.|)]?\s$/, 'numberedList' );
+			new BlockAutoformatEngine( this.editor, /^\d+[.|)]\s$/, 'numberedList' );
```

Comment:
Do we want to start with any number or as @Reinmar [sugessted](https://github.com/ckeditor/ckeditor5-autoformat/issues/42#issuecomment-346466462) from `1`? I thought it was the main idea to support only `1.` and `1)` to trigger the list.

Reply:
But now I realized that maybe we should stay what you proposed.
What if someone is under the list and want to add another item. It is intuitive to write next number in order:
![autoformat](https://user-images.githubusercontent.com/2396463/33419894-c8624fd8-d5ac-11e7-876b-b77bbc91e43e.gif)

Reply:
Good point, @szymonkups. Supporting only 1. would be unintuitive in some cases.

Reply:
I am not sure if that's the way we want to go... What about having 9 list items and writing the tenth one? We can go like this forever.

Reply:
Yeah, I know. I hoped that no one will mention this :P

TBH, it just so much doesn't matter (after all it's just a helper and in 99% of cases people will start with `1.` or apply list manually if it didn't work) that all options are fine. So, let's just accept what we have and wait.